### PR TITLE
Replace labels

### PR DESCRIPTION
**Pull request by kat1906**
- removed unnecessary label components
- fixed new line problem in text
- general tidy up

### CODE REVIEW

File: app/components/view-question-data-link/template.njk [hunk 1]
```diff
@@ -12,26 +12,26 @@
...
-        <label class='nhsuk-label'>
-          http://www.somelink.com
-        </label>
+    <div data-test-id='view-question-data-link' class='bc-c-data-link another-class'>
+      <a href='http://www.somelink.com' target='_blank'>
+        http://www.somelink.com
       </a>
     </div>
   " }}
   </code></pre>
```
Comment (alibombaye) [inline app/components/view-question-data-link/template.njk]: Remove the Generated HTML block. More effort than it's worth

File: app/components/view-question-data-text/macro.njk [hunk 2]
```diff
@@ -1,11 +1,7 @@
-{% from 'components/label/macro.njk' import label %}
-
 {% macro viewQuestionDataText(params) %}
   {% if params.questionData %}
-    <div data-test-id="view-question-data-text-{{params.questionId}}">
-      {{ label({
-        "text": params.questionData
-      }) }}
+    <div data-test-id="view-question-data-text-{{ params.questionId }}" class="bc-c-data-text {{ params.classes }}">
+      {{ params.questionData }}
```
Comment (alibombaye) [inline app/components/view-question-data-text/macro.njk]: do we not need to put it around <p> tags?

File: app/components/view-question-data-text/template.njk [hunk 3]
```diff
@@ -20,17 +21,16 @@
...
     {{- "
-      <div data-test-id='view-question-data-text-some-id'>
-        <label class='nhsuk-label'>
-          Some data
-        </label>
+      <div data-test-id='view-question-data-text-some-id' class='bc-c-data-text another-class'>
+        Some data
       </div>
     " }}
   </code></pre>
```
Comment (alibombaye) [inline app/components/view-question-data-text/template.njk]: Remove this.

File: app/components/view-question-title/template.njk [hunk 4]
```diff
@@ -10,24 +10,24 @@
...
-      <div data-test-id='view-question-title'>
-        <label class='nhsuk-label nhsuk-label--s nhsuk-u-margin-bottom-3'>
-          Title
-        </label>
-      </div>
+      <h4 data-test-id='view-question-title' class='bc-c-data-title__text another-class'>
+        Title
+      </h4>
     " }}
   </code></pre>
```
Comment (alibombaye) [inline app/components/view-question-title/template.njk]: remove

File: app/components/view-section-table-row/template.njk [hunk 5]
```diff
@@ -15,6 +15,29 @@
...
+          </div>
+          <div data-test-id='view-section-table-row-component' class='nhsuk-grid-column-one-half'>
+            <div>
+              Some inner component
+            </div>
+          </div>
+        </div>
+      </div>
+    " }}
+  </code></pre>
```
Comment (alibombaye) [inline app/components/view-section-table-row/template.njk]: delete

File: app/components/view-section-table-row/template.njk [hunk 6]
```diff
@@ -36,6 +59,26 @@
...
+            <div data-test-id='view-section-table-row-component' class='nhsuk-grid-column-two-thirds'>
+              <div>
+                Lorem ipsum dolor sit amet consectetur adipiscing elit, viverra ad torquent et lectus malesuada, natoque fermentum scelerisque urna mauris auctor. Sociis class lacinia ante per non cras aenean, morbi sem blandit praesent mus imperdiet hac, egestas diam congue donec phasellus sapien. Tempor ultricies porttitor fames potenti vestibulum maecenas montes orci elementum ligula, in id magna nascetur laoreet suspendisse ullamcorper porta justo, vivamus habitasse ridiculus eleifend taciti primis quisque consequat penatibus.
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    " }}
+  </code></pre>
```
Comment (alibombaye) [inline app/components/view-section-table-row/template.njk]: delete

File: app/sections/view-solution-description/template.njk [hunk 7]
```diff
@@ -24,48 +24,38 @@
...
+                  www.somelink.com
+                </a>
+              </div>
             </div>
           </div>
         </div>
       </div>
-    </div>
     ' }}
   </code></pre>
```
Comment (alibombaye) [inline app/sections/view-solution-description/template.njk]: Delete the Generated Html

File: package.json [hunk 8]
```diff
@@ -22,6 +22,9 @@
     "webpack": "webpack"
   },
   "author": "Fawad Ali",
+  "contributors": [
+    "Katie Armstrong (https://github.com/kat1906)"
```
Comment (alibombaye) [inline package.json]: 👍

File: app/components/view-question-data-text/macro.njk (hunk 2 above)
Comment (kat1906) [reply app/components/view-question-data-text/macro.njk]: erm what?

File: app/components/view-question-data-text/template.njk (hunk 3 above)
Comment (kat1906) [reply app/components/view-question-data-text/template.njk]: what?

File: app/components/view-section-table-row/template.njk (hunk 5 above)
Comment (kat1906) [reply app/components/view-section-table-row/template.njk]: we consistently do this elsewhere so why not here?

File: package.json (hunk 8 above)
Comment (kat1906) [reply package.json]: 😆

File: app/components/view-question-title/template.njk (hunk 4 above)
Comment (kat1906) [reply app/components/view-question-title/template.njk]: ?

**Comment by kat1906** [comment]
I have removed the generated HTML for the sections templates but kept them for the components templates

File: app/components/view-question-data-text/macro.njk (hunk 2 above)
Comment (alibombaye) [reply app/components/view-question-data-text/macro.njk]: 😂 it didn't show the `<p>` tag

File: app/components/view-question-data-text/template.njk (hunk 3 above)
Comment (alibombaye) [reply app/components/view-question-data-text/template.njk]: the generated html block in the template

File: app/components/view-section-table-row/template.njk (hunk 5 above)
Comment (alibombaye) [reply app/components/view-section-table-row/template.njk]: we decided to take it out. We've removed it elsewhere as we have gone along

File: app/components/view-section-table-row/template.njk [hunk 9]
```diff
@@ -15,6 +15,8 @@
     {% endverbatim %}
   </code></pre>
 
+  {% set commonAttributes %} data-test-id="view-section-table-row-{{ params.questionId }}" class="nhsuk-width-container bc-c-section-table-row"{%- endset -%}
```
Comment (alibombaye) [inline app/components/view-section-table-row/template.njk]: what's this about? `commonAttributes ` and where is it used?

Comment (kat1906) [reply app/components/view-section-table-row/template.njk]: oh yeah I left that there by mistake 👀 good spot

File: app/components/view-question-data-text/macro.njk (hunk 2 above)
Comment (kat1906) [reply app/components/view-question-data-text/macro.njk]: nah it is surrounded in a div

**Pull request merged**